client/my-sites/email/inbox/mailbox-selection-list/style.scss

```
@@ -0,0 +1,196 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+body {
+	background-color: var( --studio-white );
+}
+
+.mailbox-selection-list {
+
+	.mailbox-selection-list__header {
+		.formatted-header__title {
+			font-size: $font-title-large;
+			line-height: $font-headline-medium;
+
+			@include break-large {
+				font-size: $font-headline-medium;
+			}
+		}
+
+		.formatted-header__subtitle {
+			font-size: $font-body;
+			line-height: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__new-mailbox-upsell-container {
+		background: #f9f9f9;
+		align-items: center;
+		justify-content: space-between;
+		position: relative;
+		margin: 20px auto 10px;
+		padding: 10px 15px;
+		width: 240px;
+
+		@include break-medium {
+			width: 342px;
+			display: flex;
+			margin-top: 60px;
+			padding: 16px 24px;
+		}
+
+		@include break-large {
+			width: 660px;
+			display: flex;
+			margin-top: 80px;
+			padding: 16px 24px;
+		}
+
+		.mailbox-selection-list__new-mailbox-upsell-messages {
+			font-size: $font-body-small;
+			line-height: 28px;
+
+			h2 {
+				font-weight: bold;
+				font-size: $font-body;
+				line-height: 30px;
+				margin-bottom: 0;
+
+				@include break-medium {
+					margin-bottom: 8px;
+					line-height: 34px;
+				}
+
+				@include break-large {
+					margin-bottom: 8px;
+					line-height: 34px;
+				}
+			}
+		}
+
+		.mailbox-selection-list__new-mailbox-upsell-cta {
+			margin-top: 20px;
+
+			@include break-medium {
+				margin-top: 0;
+			}
+
+			@include break-large {
+				margin-top: 0;
+			}
+
+			.button {
+				font-size: $font-body-small;
+				font-weight: bold;
+			}
+		}
+	}
+
+	.mailbox-selection-list__status {
+		line-height: 24px;
+		font-size: $font-title-small;
+		position: fixed;
+		top: 50%;
+		left: 50%;
+		transform: translate( -50%, -50% );
+		display: flex;
+		align-items: center;
+		justify-content: center;
+
+		.gridicon {
+			margin-right: 14px;
+			width: 40px;
+			height: 40px;
+		}
+
+		.gridicons-cross-circle {
+			color: #ff0000;
+		}
+
+		@include break-large {
+			transform: none;
+			font-size: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__mailbox {
+		width: 310px;
+		box-shadow: none;
+		border-width: 1px 0 0;
+		border-color: #e7e9ea;
+		border-style: solid;
+		margin-bottom: 1px;
+		display: flex;
+		align-items: center;
+		padding: 12px;
+
+		@include break-medium {
+			width: 343px;
+			padding: 16px;
+		}
+
+		@include break-large {
+			width: 660px;
+		}
+
+		.gridicons-external {
+			width: 20px;
+		}
+
+		&:first-of-type {
+			border-top-width: 0;
+
+			@include break-large {
+				margin-top: 60px;
+			}
+		}
+
+		.mailbox-selection-list__domain h2 {
+			color: var( --p2-color-text );
```

Please use `--studio-wordpress-blue-100` - we don't want to be dependent on P2 style choices.
```suggestion
			color: var( --studio-wordpress-blue-100 );
```

---

TIL